### PR TITLE
add market history cache

### PR DESCRIPTION
**Pull request by ayrat555**
resolves https://github.com/poanetwork/blockscout/issues/1880
resolves https://github.com/poanetwork/blockscout/issues/2035

### CODE REVIEW

**Comment by ayrat555** [comment]
@vbaranov yes

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
 ## Current
+### Features
```
Comment (vbaranov) [inline CHANGELOG.md]: @ayrat555 there is a doubled `Features` section

Comment (ayrat555) [reply CHANGELOG.md]: fixed

**Comment by vbaranov** [comment]
@ayrat555 something wrong with gettext test

**Comment by ayrat555** [comment]
@vbaranov it was ci bug. now everything's ok

**Pull request merged**